### PR TITLE
Fix oob write in dyldcache accel loading ##crash

### PR DESCRIPTION
**Pull request by trufae**
- [ ] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

### CODE REVIEW

File: libr/bin/p/bin_dyldcache.c [hunk 1]
```diff
@@ -1230,6 +1230,10 @@ static RList *create_cache_bins(RBinFile *bf, RDyldCache *cache) {
 					ut32 k;
 					for (k = extras[j].dependentsStartArrayIndex; depArray[k] != 0xffff; k++) {
 						ut16 dep_index = depArray[k] & 0x7fff;
+						if (dep_index >= cache->hdr->imagesCount) {
+							R_LOG_ERROR ("depList contents overflow");
+							break;
```
Comment (mrmacete) [inline libr/bin/p/bin_dyldcache.c:1235]: maybe we can `continue` here, right?

Comment (trufae) [reply libr/bin/p/bin_dyldcache.c:1235]: Yeah totally, otherwise we could miss some entries

Comment (trufae) [reply libr/bin/p/bin_dyldcache.c]: Can you review and test again?

**Review by mrmacete — APPROVED** [review]
LGTM

**Pull request merged**